specification/cognitiveservices/ContentSafety/tspconfig.yaml

```diff
@@ -30,6 +30,10 @@ options:
     model-namespace: false
     flavor: azure
   "@azure-tools/typespec-ts":
+    enerateMetadata: true
```

Comment:
Note: Create or update emitter options here, SDK automation pipeline will automatically detect and generate JS SDK for us.